app/models/group.rb

```diff
@@ -27,10 +27,6 @@ class Group < ApplicationRecord
 
   validates :slug, uniqueness: { scope: :grouping }
 
-  before_validation(on: :create) do
-    create_github_team if organization
-  end
```

Comment:
This right here is what was causing flakey tests (and preventing us from creating a factorybot) I have since moved this behavior into the `Group::Creator`.